frontend/src/libs/components/button/styles.module.scss

```diff
@@ -133,6 +133,25 @@
   cursor: pointer;
 }
 
+.icon-right {
+  position: relative;
+  top: -60px;
+  left: 265px;
```

Comment:
why do we need such strict values shouldn't it be centred vertically and have something like right 10 instead of left?

Reply:
@Liza-Veis i make this trash button with position relative, so without that it won`t work

Reply:
I mean that we shouldn't use top -60 and left values here, but we should make it position absolute, center it  (something like top 50%, transformx -50%) and add right - right <padding value>px

Reply:
done